tests/test_core.py

```diff
@@ -244,6 +244,25 @@ def test_multi_file_support(self):
                     'sequence': [{'include': 'fooone'}],
                     'type': 'seq',
                 }
+            ),
+            # This tests that you can include a partial schema alongside other rules in a map
+            (
+                [
+                    self.f("partial_schemas", "7s-schema.yaml"),
+                ],
+                self.f("partial_schemas", "7s-data.yaml"),
+                {
+                    'type': 'map',
+                    'mapping': {
+                        'foo': {
+                            'type': 'str',
+                            'required': True
+                        },
+                        'bar': {
+                            'include' : 'bar'
```

Comment:
@jmccaffrey42 It looks like this space between `'include'` and `:` is breaking the CI